eyeshade/migrations/0010_snapshots/up.sql

```diff
@@ -0,0 +1,12 @@
+select execute($$
+
+insert into migrations (id, description) values ('0010', 'snapshots');
+
+create table snapshots(
+  id               uuid primary key default uuid_generate_v4(),
```

Comment:
So it seems you're merging the snapshot data together for `votes` and `transactions`. I'd suggest you split out data by its type by creating another column `kind` or `type` so when we query it'll just query the right rows rather than including excess data.

Reply:
added that feature. but pretty sure we can do that by just reaching in